core/devmode-spi/src/main/java/io/quarkus/dev/testing/ContinuousTestingWebsocketListener.java

```diff
@@ -6,7 +6,7 @@
 public class ContinuousTestingWebsocketListener {
 
     private static Consumer<State> stateListener;
-    private static volatile State lastState = new State(false, false, 0, 0, 0, 0);
+    private static volatile State lastState = new State(false, false, 0, 0, 0, 0, false, false, false);
```

Comment:
what do you think about using constants for the values ​​here?
I think it will facilitate the understanding of the code.